editor/editor_node.cpp

```diff
@@ -3836,6 +3828,14 @@ void EditorNode::_set_current_scene_nocheck(int p_idx) {
 	if (tabs_to_close.is_empty()) {
 		callable_mp(this, &EditorNode::_set_main_scene_state).call_deferred(state, get_edited_scene()); // Do after everything else is done setting up.
 	}
+
+	if (editor_data.check_and_update_scene(p_idx)) {
+		if (editor_data.get_scene_path(p_idx) != "") {
```

Comment:
```suggestion
		if (!editor_data.get_scene_path(p_idx).is_empty()) {
```
(I know it was like this originally)